kuma/urls.py

```diff
@@ -140,6 +140,9 @@ def serve_from_media_root(request, path):
     # User API service
     url('api/v1/', include('kuma.users.api.v1.urls')),
 
+    # Document API service
+    url('api/v1/', include('kuma.wiki.api.v1.urls')),
```

Comment:
A caret ``^`` is needed to restrict this to the start of the path

```suggestion
    url('^api/v1/', include('kuma.wiki.api.v1.urls')),
```

Otherwise, URLs like ``/zzz/api/v1/doc/en-US/Web`` will match. The User API line needs a similar change, to prevent https://developer.allizom.org/garbage/api/v1/whoami from matching.

Once this is done, the middleware no longer needs a special case.

Reply:
I fixed both in the latest push.